### ISSUE TITLE
darknet53,imagenet数据集上训练

### ISSUE THREAD

**Issue by njustczr** [zgcr/pytorch-ImageNet-CIFAR-COCO-VOC-training#5]
你好：
        你说多次训练会有波动，我这边darknet53在imagenet数据集上训练，现在得到最好的结果top1acc：76.5%，不知道是不是算波动范围内？我这边的训练配置和你是一样的，除了我是用分布式训练，四张卡，batchsize=124×4这个有区别吧。
我这边训练的脚本地址：https://github.com/njustczr/darknet53

**Comment by zgcr** [comment]
> 你好：
> 你说多次训练会有波动，我这边darknet53在imagenet数据集上训练，现在得到最好的结果top1acc：76.5%，不知道是不是算波动范围内？我这边的训练配置和你是一样的，除了我是用分布式训练，四张卡，batchsize=124×4这个有区别吧。
> 我这边训练的脚本地址：https://github.com/njustczr/darknet53

大的数据集波动比较小，像ImageNet上波动一般在0.1%到0.3%左右。小的数据集比如cifar100的波动区间能达到1.5%左右。 我的数据增强最后有个normalize，看你的代码里没做。另外分布式训练的点数可能会比DataParallel模式下点数略低一点。不过76.5和77的预训练模型拿去训目标检测模型的话基本上没啥差别。

**Comment by njustczr** [comment]
> normalize

normalize是有做的。。
![image](https://user-images.githubusercontent.com/30426320/93075397-4f1df480-f6b8-11ea-9001-514c7acec7d9.png)

**Comment by zgcr** [comment]
> > normalize
> 
> normalize是有做的。。
> ![image](https://user-images.githubusercontent.com/30426320/93075397-4f1df480-f6b8-11ea-9001-514c7acec7d9.png)

ok，训到这个点去做目标检测已经不怎么影响点数了，0.5个百分点在目标检测模型训练时基本上没有什么区别。

**Comment by njustczr** [comment]
> > > normalize
> > 
> > 
> > normalize是有做的。。
> > ![image](https://user-images.githubusercontent.com/30426320/93075397-4f1df480-f6b8-11ea-9001-514c7acec7d9.png)
> 
> ok，训到这个点去做目标检测已经不怎么影响点数了，0.5个百分点在目标检测模型训练时基本上没有什么区别。

好的，多谢。
